docs/src/pages/AlignmentPage/AlignmentPage.js

```diff
@@ -279,7 +279,9 @@ const blurb = [
   </Paragraph>,
 ];
 
-const AlignmentSubPage = () => <WebComponentPage examples={components} />;
+const AlignmentSubPage = () => (
+  <WebComponentPage examples={components} packageName="bpk-component-icon" />
```

Comment:
Is this because alignment lives inside `bpk-component-icon`?

Reply:
Yeah. I thought about making it point to the specific `withAlignment.js` file, but that would have required a lot of changes on `WebComponentPage.js` and it didn't feel worth it.